server/src/test/java/com/linecorp/centraldogma/server/internal/api/converter/WatchRequestConverterTest.java

```diff
@@ -51,6 +51,36 @@ void convertWatchRequest() throws Exception {
         assertThat(watchRequest.timeoutMillis()).isEqualTo(10000); // 10 seconds
     }
 
+    @Test
+    void convertRequestWithETaggedIfNoneMatchHeaderToWatchRequest() throws Exception {
+        final ServiceRequestContext ctx = mock(ServiceRequestContext.class);
+        final AggregatedHttpRequest request = mock(AggregatedHttpRequest.class);
+        final RequestHeaders headers = RequestHeaders.of(HttpMethod.GET, "/",
+                                                         HttpHeaderNames.IF_NONE_MATCH, "\"-1\"",
```

Comment:
Could you also add cases that are not valid? e.g. `"-1`, `w"-1` etc.

Reply:
nit: We can create an `HttpRequest` without using `mock`:
```java
HttpRequest request = HttpRequest.of(headers);
ServiceRequestContext.of(request);
```

Reply:
`convertRequest` requires `AggregatedHttpRequest` and `ServiceRequestContext`. `ServiceRequestContext` requires `HttpRequest`. Is there a way creating `ServiceRequestContext` with `AggregatedHttpRequest`?

Reply:
Hmm... Seems like it..+_+
```java
final HttpRequest request = HttpRequest.of(headersWithSmallCase);
final ServiceRequestContext serviceRequestContext = ServiceRequestContext.of(request);
final WarchRequest warchRequest = convert(ctx, (AggregatedHttpRequest) request);
```

Reply:
We can do:
```java
final AggregatedHttpRequest request = AggregatedHttpRequest.of(headers);
final ServiceRequestContext ctx = ServiceRequestContext.of(request.toHttpRequest());
```
When we implement this class, we didn't have `ServiceRequestContext.of(...)` so we used mock. 😅 
Sorry about it, and thanks for changing it. 😆

Reply:
Thank you 😄 I applied it.